packages/unity-bootstrap-theme/src/scss/extends/_charts-and-graphs.scss

```diff
@@ -34,6 +34,15 @@
   }
 }
 
+.uds-charts-and-graphs-overlay {
+  @extend %shared-styles;
+  justify-content: center;
+}
+
+.uds-charts-and-graphs-overlay p {
+  @extend %shared-styles;
+}
```

Comment:
Update looks good, can we add margin-bottom: 0; as we do not need this for this `p` tag and the margin pushes the text up.

Reply:
Done @scott-williams-az